charters/charter-template.md

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Once there's agreement on the template outline, I will add detailed descriptions for each area.

Reply:
Looks good to me - my only comment is that some non-technical projects might benefit from a charter too, so we might want to "genericize" the last couple of steps here. For example, if we decide to create a DCP communications team it may be helpful to write up a charter for it.

Reply:
This process is slightly different then what was presented in the Appendix of the PM Retrospective presentation. https://docs.google.com/presentation/d/1c2pfeck-b79Pjpo0-_E-2pKnePdX38XoDnezOryEFCo/edit#slide=id.g3c820c8e83_0_58 . I am reading this as the technical architecture team always approves all charters for the HCA DCP. This is different in that the Appendix where the PM++ or technical architecture team could approve depending on the nature of the charter. If I read this correctly, what motivates this change?

Also, is it the responsibility of the approver to determine if the project is against HCA governance? From the intro of your slides, I can appreciate the want to focus on processes and decisions within the DCP scope but that creates an incomplete system in my mind that is detached from our first purpose, to make the DCP for the HCA project.

Reply:
@jodihir and @TimothyTickle - regarding your comments on charters for _non-technical projects_ , I intentionally did not include a path for governance/process charters to be approved by DCP PM, as originally proposed in Appendix A of the PM Retrospective. The change was motivated by conversations at the meetings where strong preferences were shared to keep the charter process simple and focused on software/technical charters.

 

The original sketch was an experiment to see if there was a clear demand for such charters with specific plans to write ​(more than?) ​one. It’s simple for me to add a “process” charter to the workflow with approval by DCP PM, but I would prefer the change to be prompted by requirements rather than “completeness”.

Reply:
@TimothyTickle - regarding _is it the responsibility of the approver to determine if the project is against HCA governance?_ - could you offer a realistic and concrete example to help my understanding? In general, I feel like the DCP community would never allow such a proposed charter to demonstrate consensus and reach Architecture for approval.

Reply:
**Charters for non-technical projects**
I am ok with that, we can wait to when we need it. That being said, it should be clear that these are technical charters; what it means to be a techinical charter and what kinds of things this covers. And I think that definition should in some way be colocated with the technical charter template.

Reply:
**HCA Governance Alignment**
I think it has not happened yet but has almost happened several times. It often arises when user requests are not fully aligned to HCA governance and we are split between the two. I am just struggling to understand how this is addressed not implicitly but with a specific point in the process.

Reply:
@TimothyTickle - I can understand HCA Governance concerns for rfcs, but less so for charters which is why I was curious if there was a specific example in mind. My expectation is that such cases would be immediately flagged during community review - for example, by you.  We could add a statement that _Science PM_ (Jonah) would review for alignment with HCA Science governance.

Reply:
@brianraymor as I said when we spoke I think limiting charters to technical issues isn't the best plan as even with good oversight there are clear non-technical components such as data wrangling or website content where having a similar set of goals, scope, and milestones defined would be useful. I appreciate some things are perhaps a bit fuzzier in these cases but not too much more. 

I am going to work with @claymfischer to write both the metadata and support charters so hopefully we can use that as a launch point for the non-technical charters

Reply:
+1 from me on charters not being purely limited to technical. I can imagine very clear scenarios where technical components will benefit from having a view over data wrangling goals in terms of scaling and new data types, for example. It seems clear to me though that the technical charters are the more pressing concern

Reply:
I am not sure I fully understand how we are going to leverage charters. In that light, experimenting with technical and nontechnical seems appropriate where people feel they can apply them.